### PR TITLE
UI changes 

### PR DESCRIPTION
**Pull request by manojnagar**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -68,6 +68,7 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+
```
Comment (manojnagar) [inline app/src/main/AndroidManifest.xml]: Do not add extra spacing in any files

**Comment by nimeshkverma** [comment]
@manojnagar 

Merge kae dena dekh ke!

File: app/src/main/java/com/goupwards/AadharNumberWatcher.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package com.goupwards;
+
+import android.text.Editable;
+import android.text.TextWatcher;
+
+/**
+ * Created by Vineet on 1/30/2017.
+ */
+
+public class AadharNumberWatcher implements TextWatcher {
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/AadharNumberWatcher.java]: Why need this watcher? can you add me detail which format you want

File: app/src/main/java/com/goupwards/KYC_Aadhar_Screen.java [hunk 3]
```diff
@@ -58,7 +58,6 @@ public static boolean validateAadharNumber(String aadharNumber){
                 Log.d("Validate ","is"+isValidAadhar);
             }
             return isValidAadhar;
-
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/KYC_Aadhar_Screen.java]: what you are working on KYC screen? first fininsh eligibility check

File: app/src/main/java/com/goupwards/Lin.java [hunk 4]
```diff
@@ -74,9 +82,23 @@ public boolean shouldOverrideUrlLoading(final WebView view, final String url) {
 
         @Override
         public void onPageFinished (WebView view, String url){
+//            url.matches
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/Lin.java]: remove unused code

File: app/src/main/java/com/goupwards/Lin.java [hunk 5]
```diff
@@ -74,9 +82,23 @@ public boolean shouldOverrideUrlLoading(final WebView view, final String url) {
...
+//            if(url.contains((String.valueOf(200)));
             view.loadUrl("javascript:alert(functionThatReturnsSomething())");
         }
         @Override
+        public void onReceivedSslError(final WebView view, final SslErrorHandler handler, final SslError error) {
+            // TODO Auto-generated method stub
+            super.onReceivedSslError(view, handler, error);
+            // handle your SSL related error here
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/Lin.java]: remove unused code, never add dead code in branch

File: app/src/main/java/com/goupwards/Singleton.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
+package com.goupwards;
+
+/**
+ * Created by Vineet on 1/28/2017.
+ */
+
+public class Singleton  {
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/Singleton.java]: remove this class, if require write proper reason

File: app/src/main/java/com/goupwards/checkcreditflow/contract/EmploymentDetailContract.java [hunk 7]
```diff
@@ -8,6 +8,8 @@
 
 public class EmploymentDetailContract {
   public interface View {
+
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/checkcreditflow/contract/EmploymentDetailContract.java]: do not add extra lines

File: app/src/main/java/com/goupwards/checkcreditflow/view/EducationalQualificationFragment.java [hunk 8]
```diff
@@ -21,6 +22,9 @@ protected int getLayoutResourceId() {
     protected void initViews(View rootView) {
         presenter = new EducationalQualificationPresenter(this);
         rootView.findViewById(R.id.home_save_and_proceed_button).setOnClickListener(this);
+        TextView passingyear= (TextView)rootView.findViewById(R.id.edu_passing_year);
+        String pasingyear= passingyear.getText().toString();
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/checkcreditflow/view/EducationalQualificationFragment.java]: get a method in View as getPassingYead and return value, directly do not call presenter method to set values

File: app/src/main/java/com/goupwards/checkcreditflow/view/EducationalQualificationFragment.java [hunk 9]
```diff
@@ -21,6 +22,9 @@ protected int getLayoutResourceId() {
...
+        presenter.passingYearCheck(pasingyear);
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/checkcreditflow/view/EducationalQualificationFragment.java]: remove this and create gettes as above comment

File: app/src/main/java/com/goupwards/checkcreditflow/view/EmploymentDetailFragment.java [hunk 10]
```diff
@@ -37,8 +40,10 @@ protected void initViews(View rootView) {
     companyname = (EditText) rootView.findViewById(R.id.prof_company_name_edit_text);
     officeemail = (EditText) rootView.findViewById(R.id.prof_office_email_edit_text);
     income = (EditText) rootView.findViewById(R.id.prof_income_edit_text);
+    working = (EditText)rootView.findViewById(R.id.prof_workingsince);
+    working.setOnClickListener(this);
     displaytext = (TextView) rootView.findViewById(R.id.displaytext);
-    rootView.findViewById(R.id.prof_join_date).setOnClickListener(this);
+//    rootView.findViewById(R.id.prof_join_date).setOnClickListener(this);
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/checkcreditflow/view/EmploymentDetailFragment.java]: remove dead code

File: app/src/main/java/com/goupwards/checkcreditflow/view/EmploymentDetailFragment.java [hunk 11]
```diff
@@ -51,6 +56,9 @@ public void onClick(View v) {
       case R.id.prof_save_and_proceed_button:
         presenter.onSaveAndProceedButtonClicked();
         break;
+      case R.id.prof_workingsince:
+
+        presenter.onJoinDataButtonClicked();
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/checkcreditflow/view/EmploymentDetailFragment.java]: add break  and remove extra line

File: app/src/main/java/com/goupwards/checkcreditflow/view/MiscellaneousFragment.java [hunk 12]
```diff
@@ -43,8 +43,9 @@ public void onClick(View v) {
 
     @Override
     public void openLinkedInSignInFlow() {
-        Intent intent = new Intent(getContext(), Lin.class);
+        Intent intent = new Intent(getActivity(), Lin.class);
         startActivity(intent);
+        // null pointer here
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/checkcreditflow/view/MiscellaneousFragment.java]: remove dead code

File: app/src/main/java/com/goupwards/flashscreen/FlashPresenter.java [hunk 13]
```diff
@@ -22,6 +22,12 @@ public FlashPresenter(FlashContract.View flashView) {
   }
 
   @Override
+  public void flashAnimation(){
+
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/flashscreen/FlashPresenter.java]: add TODO statement if pending for implementation

File: app/src/main/java/com/goupwards/flashscreen/FlashScreenActivity.java [hunk 14]
```diff
@@ -1,24 +1,33 @@
...
 import android.os.Bundle;
+import android.view.animation.Animation;
+import android.view.animation.AnimationUtils;
+import android.widget.ImageView;
 
 import com.goupwards.R;
 import com.goupwards.homescreen.HomeEligibilityActivity;
 import com.goupwards.ui.BaseActivity;
 
+
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/flashscreen/FlashScreenActivity.java]: why you are adding so many lines in code

File: app/src/main/java/com/goupwards/homescreen/HomeEligibilityActivity.java [hunk 15]
```diff
@@ -13,16 +16,27 @@
 public class HomeEligibilityActivity extends BaseActivity implements HomeEligibilityContract.View, View.OnClickListener {
 
   private HomeEligibilityContract.Actions presenter;
-  private TextView textViewUser;
+  private TextView textViewUser, sucess1, percentcomplete;
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/homescreen/HomeEligibilityActivity.java]: give proper names as successTextVIew etc

File: app/src/main/java/com/goupwards/homescreen/HomeEligibilityActivity.java [hunk 16]
```diff
@@ -40,6 +55,9 @@ public void onClick(View view) {
       case R.id.check_eligibility_layout:
         presenter.onCheckEligibilityLayoutClicked();
         break;
+        case R.id.homekycclick:
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/homescreen/HomeEligibilityActivity.java]: code fomatting

File: app/src/main/java/com/goupwards/homescreen/HomeEligibilityActivity.java [hunk 17]
```diff
@@ -60,6 +78,12 @@ public void openPanVerifyActivity() {
   public void setUserName(String displayMessage) {
     textViewUser.setText(displayMessage);
   }
+
+    @Override
+    public void setEligibilityProgressBar(int progress) {
+        progressBar.setProgress(progress);
+        percentcomplete.setText(progress+"% Complete");
```
Comment (manojnagar) [inline app/src/main/java/com/goupwards/homescreen/HomeEligibilityActivity.java]: Create string in string.xml and use via passing value

File: app/src/main/res/layout/activity_demo.xml [hunk 18]
```diff
@@ -36,4 +36,10 @@
         android:layout_height="wrap_content"
         android:layout_marginLeft="@dimen/normal"
         android:text="KYC Flow"/>
+
+    <Button
+        android:text="Buttonkyv"
+        android:layout_width="241dp"
```
Comment (manojnagar) [inline app/src/main/res/layout/activity_demo.xml]: create constant in dimen.xml with proper names

File: app/src/main/res/layout/activity_flash_screen_layout.xml [hunk 19]
```diff
@@ -1,7 +1,15 @@
...
               android:layout_width="match_parent"
               android:layout_height="match_parent"
-              android:orientation="vertical">
+              android:orientation="vertical"
+    android:background="@drawable/color_gradient">
 
-</LinearLayout>
\ No newline at end of file
+    <ImageView
+        android:layout_width="80dp"
```
Comment (manojnagar) [inline app/src/main/res/layout/activity_flash_screen_layout.xml]: create a constant in dimen, do not hardcode any vlaues

File: app/src/main/res/layout/fragment_employment_detail_layout.xml [hunk 20]
```diff
@@ -75,54 +140,160 @@
...
+                    android:inputType="number"/>
+                    <TextView
+                        android:layout_marginTop="5dp"
+                        android:textColor="@color/colorWhite"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="Years"/>
+                    <android.support.design.widget.TextInputEditText
+                        android:id="@+id/prof_office_wrkexp"
+                        android:layout_width="50dp"
```
Comment (manojnagar) [inline app/src/main/res/layout/fragment_employment_detail_layout.xml]: Check other hardcoding values

File: model/src/main/java/com/goupwards/home/HomeResponse.java [hunk 21]
```diff
@@ -39,6 +35,24 @@ public HomeCardInfo getKycCard() {
...
     }
-  }
 
+    public HomeCardInfo getEligibilityCard() {
+      return homeFormData.eligibilityCard;
+    }
+
+//    public class HomeCardInfo {
+//      @SerializedName("title")
+//      private HomeCardTitle homeCardTitle;
```
Comment (manojnagar) [inline model/src/main/java/com/goupwards/home/HomeResponse.java]: remove dead code from all project and do not modify model, storage and network module. Remove all changes you made

**Pull request merged**